test/golden/library_example.golden

```diff
@@ -38,79 +38,9 @@
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/ListBooksRequest.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/ListBooksRequestOrBuilder.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/ListBooksResponse.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/ListBooksResponseOrBuilder.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/ListShelvesRequest.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/ListShelvesRequestOrBuilder.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/ListShelvesResponse.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/ListShelvesResponseOrBuilder.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/MergeShelvesRequest.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/MergeShelvesRequestOrBuilder.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/MoveBookRequest.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/MoveBookRequestOrBuilder.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/Shelf.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/ShelfName.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/ShelfOrBuilder.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/UpdateBookRequest.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/java/com/google/example/library/v1/UpdateBookRequestOrBuilder.java
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/src/main/proto/google/example/library/v1/library.proto
-/java/gapic-google-cloud-library_example-v1/proto-google-cloud-library_example-v1/templates/apidocs_index.html.template
-/java/gapic-google-cloud-library_example-v1/settings.gradle
```

Comment:
Why is the settings.gradle disappearing?

Reply:
Because it is not needed. The current (before this PR) set of files (content of library_example.golden) is a Frankenstein versions of lightweight jars (as we deploy it in maven) and fat jar, where one partially overwrites another. After this PR there is no fat jar, and setings.gradle is a part of that fat jar (note that settings.gralde is under gapic-<client> folder, when in api-clien-staging settings.gradle is in root directory)

Reply:
Sounds good.